Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM --platform=$BUILDPLATFORM ubuntu as build
+ENV HOME="/root"
+WORKDIR $HOME
+
+RUN apt update && apt install -y build-essential curl python3-venv
```

Comment:
Probably still good to follow best practice to use `apt-get`, clean the cache, not install recommends.

Even if it's not in the production images, reducing the size of the cached layers is important.